backend/src/Api/Console/Controller/ProjectController.php

```diff
@@ -37,4 +43,19 @@ public function getNewsletterById(Project $project): JsonResponse
     {
         return $this->json(new ProjectObject($project));
     }
-}
\ No newline at end of file
+
+    #[Route('/project', methods: 'PATCH')]
+    #[NoScopeRequired]
```

Comment:
I feel like this was a mistake. It's better to add new scopes: `project.read`, `project.write` and use them. Then remove the NoScopeRequired attribute